examples/tutorials/vectors.py

```diff
@@ -0,0 +1,461 @@
+# Using ``pygmt.config(PROJ_LENGTH_UNIT="i")``, the default unit
+# can be changed to inches in the second plotted vector.
+
+fig = pygmt.Figure()
+# Vector 1 with default unit as cm
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X5i/5i",
+    frame="a",
+    x=2,
+    y=8,
+    direction=[[0], [3]],
+    style="v1c+e",
+    pen="2p",
+    color="red3",
+)
+# Vector 2 after changing default unit to in
+with pygmt.config(PROJ_LENGTH_UNIT="i"):
+    fig.plot(
+        x=2,
+        y=7,
+        direction=[[0], [3]],
+        style="v1c+e",
+        pen="2p",
+        color="red3",
+    )
+fig.show()
+
+########################################################################################
+# Vectors can also be plotted by including all the information
+# about a vector in a single list. However, this requires creating
+# a list for all vectors and passing it into a numpy array object.
+# Each vector list contains the information structured as:
+# ``[x_start, y_start, direction_degrees, length]``
+#
+# If this approach is chosen, the ``data`` parameter must be
+# used instead of ``x``, ``y`` and  ``direction``.
+
+# Create a list of lists that include each vector information
+vectors = np.array([[2, 3, 45, 4]])  # vectors structure: [[ 2  3 45  4]]
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    data=vectors,
+    style="v0.6c+e",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# Using the functionality mentioned in the previous example,
+# multiple vectors can be plotted at the same time. Another
+# vector could be simply added to the 2d numpy array object
+# and passed using ``data`` parameter.
+
+# vector specifications structured as: [x_start, y_start, direction_degrees, length]
+vector_1 = [2, 3, 45, 4]
+vector_2 = [7.5, 8.3, -120.5, 7.2]
+# Create a list of lists that include each vector information
+vectors = np.array([vector_1] + [vector_2])
+# vectors structure:
+# [[   2.     3.    45.     4. ]
+#  [   7.5    8.3 -120.5    7.2]]
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    data=vectors,
+    style="v0.6c+e",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# In this example, cartesian vectors are plotted over a Mercator
+# projection of the continental US. The x values represent the
+# longitude and y values represent the latitude where the vector starts.
+
+# create a plot with coast, Mercator projection (M) over the continental US
+fig = pygmt.Figure()
+fig.coast(
+    region=[-127, -64, 24, 53],
+    projection="M10c",
+    frame="ag",
+    borders=1,
+    area_thresh=4000,
+    shorelines="0.25p,black",
+    land="grey",
+    water="lightblue",
+)
```

Comment:
`borders` and `shorelines` are used plot lines, while `area_thresh`, `land`, `water` are related to area.
```suggestion
    borders=1,
    shorelines="0.25p,black",
    area_thresh=4000,
    land="grey",
    water="lightblue",
)
```